metadata/content.go

```diff
@@ -103,10 +107,13 @@ func (cs *contentStore) Update(ctx context.Context, info content.Info, fieldpath
 	}
 	if err := update(ctx, cs.db, func(tx *bolt.Tx) error {
 		bkt := getBlobBucket(tx, ns, info.Digest)
+		if bkt == nil {
```

Comment:
Implicitly updating the images in the shared namespace looks scary for me.

Reply:
I can see that. I took this out as I don't think it's necessary for sharing.

Reply:
hmm, I guess this didn't get removed; see #5807 bullet 2. Can be a follow-up PR.

Reply:
iirc I did remove it but it is actually necessary for this to work. I have seen #5807 though and am will engage there to see if I can understand the best method to improve this.